library/alloc/tests/vec.rs

```diff
@@ -2141,4 +2267,4 @@ fn test_extend_from_within_panicing_clone() {
     std::panic::catch_unwind(move || vec.extend_from_within(..)).unwrap_err();
 
     assert_eq!(count.load(Ordering::SeqCst), 4);
-}
+}
```

Comment:
Should have a trailing newline.

Reply:
i think `x.py fmt` removes it

Reply:
Huh, that's surprising. Would be the first time I see it doing so.